File: Mdc.java
import java.util.Scanner;

public class Mdc {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Digite o primeiro numero: ");
        int num1 = sc.nextInt();

        System.out.print("Digite o segundo numero: ");
        int num2 = sc.nextInt();

        sc.close();

        int mdc = MDC(num1, num2);

        System.out.println("O MDC dos numeros escolhidos eh " + mdc + ".");
    }

    public static int MDC(int num1, int num2) {
        while (num2 != 0) {
            int temp = num2;
            num2 = num1 % num2;
            num1 = temp;
        }
        return num1;
    }
}

File: mdc.js
function calcularMDC() {

    let num1 = parseInt(document.getElementById("num1").value);
    let num2 = parseInt(document.getElementById("num2").value);

    if (isNaN(num1) || isNaN(num2)) {
        alert("Por favor, insira números válidos.");
        return;
    }

    let mdc = calcular(num1, num2);

    document.getElementById("resultado").textContent = `O MDC dos números escolhidos é ${mdc}.`;
}

function calcular(num1, num2) {
    while (num2 !== 0) {
        let temp = num2;
        num2 = num1 % num2;
        num1 = temp;
    }
    return num1;
}